handlers/main.yml

```diff
@@ -1,7 +1,11 @@
 ---
+- name: supervisord update consul-alerts
+  command: supervisorctl update consul_alerts
+  notify: supervisord restart consul-alerts
+  when: consul_alerts_supervisor_enabled
+
 - name: supervisord restart consul-alerts
   supervisorctl:
-    name: 'consul_alerts:*'
+    name: 'consul_alerts:'
```

Comment:
Ah, I see. I didn't realize that. I thought the quotes should be enough to protect the value from being interpolated by Ansible. Nice find :smile_cat: